### ISSUE TITLE
[question] It is possible to disable some parsers

### ISSUE THREAD

**Issue by Blaztix** [cebe/markdown-latex#15]
I am coding a write section, and I dont want users can uses some markdown features, I dont want reinvet the wheel and for this I am trying with markdown.

I hope can uses only bolds, italics, lists, headers. 

And disable codes, quoutes, links and images.

Its posible with this library? Or I need some different

**Comment by cebe** [comment]
yes, it is possible, check out the documentation at https://github.com/cebe/markdown#composing-your-own-markdown-flavor for the latex variant of the parser that requires a bit more work, but it is still possible.
